packages/laconia-event/src/apigateway/ApiGatewayInputHeaders.js

```diff
@@ -4,5 +4,13 @@ module.exports = class ApiGatewayInputHeaders {
     Object.entries(eventHeaders).forEach(([headerName, headerValue]) => {
```

Comment:
We shouldn't need the line above anymore i.e. `Object.assign(this, eventHeaders);`